### ISSUE TITLE
mousepad doesn't work after update to gnome-45

### ISSUE THREAD

**Issue by frosth555** [GSConnect/gnome-shell-extension-gsconnect#1693]
### Describe the bug

after gnome update mouspad stops working. 
coursor doesn't move at all or move a little bit sometimes.  
clicking (or double/triple) or typing doesn't work too. 

### GSConnect version

git commit: 018c7fe6172231eb27b7dc2f640eccd229c7319c

### Installed from

GNOME Extensions website

### GNOME Shell version

45.0

### Linux distribution/release

archlinux

### Paired device(s)

samsung s21

### KDE Connect app version

1.29.0

### Support log

```plain text
GSConnect: 55 (user)
GJS:       17800
Session:   wayland
OS:        Arch Linux
--------------------------------------------------------------------------------
paź 20 16:28:23 systemd[1]: Starting Cleanup of Temporary Directories...
paź 20 16:28:23 systemd[1]: systemd-tmpfiles-clean.service: Deactivated successfully.
paź 20 16:28:23 systemd[1]: Finished Cleanup of Temporary Directories.
paź 20 16:28:48 gjs[1989]: ../glib/gobject/gsignal.c:2777: instance '0x55ac7815e100' has no handler with id '53849'
paź 20 16:28:48 gjs[1989]: ../glib/gobject/gsignal.c:2777: instance '0x55ac7815e100' has no handler with id '51400'
paź 20 16:28:48 gjs[1989]: ../glib/gobject/gsignal.c:2777: instance '0x55ac78e44eb0' has no handler with id '7152'
paź 20 16:28:48 gjs[1989]: ../glib/gobject/gsignal.c:2777: instance '0x55ac78850ea0' has no handler with id '15319'
paź 20 16:28:48 gjs[1989]: ../glib/gobject/gsignal.c:2777: instance '0x55ac784c59d0' has no handler with id '6829'
paź 20 16:28:48 gjs[1989]: ../glib/gobject/gsignal.c:2777: instance '0x55ac784c59d0' has no handler with id '6833'
paź 20 16:28:48 gjs[1989]: ../glib/gobject/gsignal.c:2777: instance '0x55ac794f2a50' has no handler with id '14995'
paź 20 16:28:48 gjs[1989]: ../glib/gobject/gsignal.c:2777: instance '0x55ac794f2a50' has no handler with id '14999'
paź 20 16:28:48 gjs[1989]: ../glib/gobject/gsignal.c:2777: instance '0x55ac787fb280' has no handler with id '32099'
paź 20 16:28:48 gjs[1989]: ../glib/gobject/gsignal.c:2777: instance '0x55ac78888c30' has no handler with id '31777'
paź 20 16:28:48 gjs[1989]: ../glib/gobject/gsignal.c:2777: instance '0x55ac78888c30' has no handler with id '31781'
paź 20 16:28:54 gjs[2656]: GFileInfo created without standard::symlink-target
paź 20 16:28:54 gjs[2656]: file ../glib/gio/gfileinfo.c: line 2062 (g_file_info_get_symlink_target): should not be reached
paź 20 16:28:54 systemd[823]: app-gnome-org.gnome.Shell.Extensions.GSConnect.Preferences-2651.scope: Couldn't move process 2651 to requested cgroup '/user.slice/user-1000.slice/user@1000.service/app.slice/app-gnome-org.gnome.Shell.Extensions.GSConnect.Preferences-2651.scope': No such process
paź 20 16:28:54 systemd[823]: app-gnome-org.gnome.Shell.Extensions.GSConnect.Preferences-2651.scope: Failed to add PIDs to scope's control group: No such process
paź 20 16:28:54 systemd[823]: app-gnome-org.gnome.Shell.Extensions.GSConnect.Preferences-2651.scope: Failed with result 'resources'.
paź 20 16:28:54 systemd[823]: Failed to start Application launched by gnome-shell.
paź 20 16:28:54 dbus-daemon[615]: [system] Activating via systemd: service name='org.freedesktop.hostname1' unit='dbus-org.freedesktop.hostname1.service' requested by ':1.80' (uid=1000 pid=2656 comm="gjs /home/arek/.local/share/gnome-shell/extensions")
paź 20 16:28:54 systemd[1]: Starting Hostname Service...
paź 20 16:28:54 dbus-daemon[615]: [system] Successfully activated service 'org.freedesktop.hostname1'
paź 20 16:28:54 systemd[1]: Started Hostname Service.
paź 20 16:29:04 systemd[823]: Started Application launched by gnome-shell.
paź 20 16:29:04 dbus-daemon[845]: [session uid=1000 pid=845] Activating via systemd: service name='org.gnome.Terminal' unit='gnome-terminal-server.service' requested by ':1.111' (uid=1000 pid=2698 comm="/usr/bin/gnome-terminal")
paź 20 16:29:04 systemd[823]: Starting GNOME Terminal Server...
paź 20 16:29:04 dbus-daemon[845]: [session uid=1000 pid=845] Successfully activated service 'org.gnome.Terminal'
paź 20 16:29:04 systemd[823]: Started GNOME Terminal Server.
paź 20 16:29:04 gnome-shell[908]: meta_window_set_stack_position_no_sync: assertion 'window->stack_position >= 0' failed
paź 20 16:29:04 systemd[823]: Started VTE child process 2726 launched by gnome-terminal-server process 2705.
paź 20 16:29:24 systemd[1]: systemd-hostnamed.service: Deactivated successfully.
paź 20 16:29:37 org.gnome.Shell.Extensions.GSConnect[2201]: [/service/device.js:_readLoop:364]: Galaxy S21 5G: {
                                                               "id": 1697812176483,
                                                               "type": "kdeconnect.mousepad.request",
                                                               "body": {
                                                                 "dx": 0,
                                                                 "dy": 0
                                                               }
                                                             }
paź 20 16:29:37 org.gnome.Shell.Extensions.GSConnect[2201]: [/service/device.js:_readLoop:364]: Galaxy S21 5G: {
                                                               "id": 1697812176490,
                                                               "type": "kdeconnect.mousepad.request",
                                                               "body": {
                                                                 "dx": -1,
                                                                 "dy": 1
                                                               }
                                                             }
paź 20 16:29:37 org.gnome.Shell.Extensions.GSConnect[2201]: [/service/components/input.js:_ensureAdapter:431]: "Creating Mutter RemoteDesktop session"
paź 20 16:29:37 org.gnome.Shell.Extensions.GSConnect[2201]: [/service/components/input.js:movePointer:480]: this._session is null
                                                             movePointer@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/components/input.js:480:13
                                                             _handleInput@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/plugins/mousepad.js:206:29
                                                             handlePacket@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/plugins/mousepad.js:172:22
                                                             handlePacket@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/device.js:441:25
                                                             _readLoop@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/device.js:365:22
                                                             async*setChannel@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/device.js:346:18
                                                             _onChannel@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/manager.js:229:20
                                                             channel@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/core.js:436:19
                                                             _onIdentity@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/backends/lan.js:390:18
                                                             async*_onIncomingIdentity@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/backends/lan.js:344:18
                                                             @/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/daemon.js:725:17
paź 20 16:29:37 org.gnome.Shell.Extensions.GSConnect[2201]: [/service/device.js:_readLoop:364]: Galaxy S21 5G: {
                                                               "id": 1697812176499,
                                                               "type": "kdeconnect.mousepad.request",
                                                               "body": {
                                                                 "dx": -1,
                                                                 "dy": 1
                                                               }
                                                             }
paź 20 16:29:37 org.gnome.Shell.Extensions.GSConnect[2201]: [/service/components/input.js:movePointer:480]: this._session is null
                                                             movePointer@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/components/input.js:480:13
                                                             _handleInput@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/plugins/mousepad.js:206:29
                                                             handlePacket@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/plugins/mousepad.js:172:22
                                                             handlePacket@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/device.js:441:25
                                                             _readLoop@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/device.js:365:22
                                                             async*setChannel@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/device.js:346:18
                                                             _onChannel@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/manager.js:229:20
                                                             channel@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/core.js:436:19
                                                             _onIdentity@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/backends/lan.js:390:18
                                                             async*_onIncomingIdentity@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/backends/lan.js:344:18
                                                             @/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/daemon.js:725:17
paź 20 16:29:37 org.gnome.Shell.Extensions.GSConnect[2201]: [/service/device.js:_readLoop:364]: Galaxy S21 5G: {
                                                               "id": 1697812176507,
                                                               "type": "kdeconnect.mousepad.request",
                                                               "body": {
                                                                 "dx": -1,
                                                                 "dy": 2
                                                               }
                                                             }
paź 20 16:29:37 org.gnome.Shell.Extensions.GSConnect[2201]: [/service/components/input.js:movePointer:480]: this._session is null
                                                             movePointer@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/components/input.js:480:13
                                                             _handleInput@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/plugins/mousepad.js:206:29
                                                             handlePacket@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/plugins/mousepad.js:172:22
                                                             handlePacket@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/device.js:441:25
                                                             _readLoop@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/device.js:365:22
                                                             async*setChannel@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/device.js:346:18
                                                             _onChannel@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/manager.js:229:20
                                                             channel@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/core.js:436:19
                                                             _onIdentity@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/backends/lan.js:390:18
                                                             async*_onIncomingIdentity@/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/backends/lan.js:344:18
                                                             @/home/arek/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/service/daemon.js:725:17
paź 20 16:29:37 org.gnome.Shell.Extensions.GSConnect[2201]: [/service/device.js:_readLoop:364]: Galaxy S21 5G: {
                                                               "id": 1697812176516,
                                                               "type": "kdeconnect.mousepad.request",
                                                               "body": {
                                                                 "dx": -5,
                                                                 "dy": 5
                                                               }
                                                             }

and again and again...
```

### Notes

what was done: removing  app and settings on gnome/android, 
I checked on fedora, problem persists.

**Comment by ElonSatoshi** [comment]
https://github.com/andyholmes/valent/issues/587 possibly related

**Comment by andyholmes** [comment]
Should be closed by #1699
